Gemfile

```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-gem "jekyll", "~> 3.8.5"
+# gem "jekyll", "~> 4.2.2"
```

Comment:
Shouldn't this be `gem "jekyll", "~> 3.9.2"`?

Reply:
Yes, this has been corrected!